episodes/60-wrap-up.md

```diff
@@ -141,9 +138,6 @@ Below are some additional resources to help you continue learning:
 
 - [Additional episode on persisting data](../learners/persistence.md)
 - [Additional episode on databases](../learners/databases.md)
-- [CodeRefinery courses on FAIR
-  (Findable, Accessible, Interoperable, and Reusable)
-  software practices][coderefinery-lessons]
```

Comment:
Any reason for removing this link to external resources?

Reply:
Yes, I realised that this course is more advanced compared to the FAIR ones (I also initially added the SSI FAIR course then decided to remove both). We can put the CodeRefinery one back - but it is not really a follow-up reading. Not really sure how to deal with this - maybe reword how we present the CodeRefinery lesson (having looked at them they are not mentioning FAIR any more).

Reply:
I put back the link to CodeRefinery but with different link text - see [d415de8](https://github.com/bielsnohr/python-intermediate-development/commit/d415de8c8763e4ee44e88c2153964b73dd938fe7). They are a great resource to mention even though we cover all (or most of) these topics ourselves.

Reply:
This is a sensible solution and compromise. Thanks!